MapboxMobileEvents/MMEEvent.m

```diff
@@ -5,6 +5,20 @@
 
 @implementation MMEEvent
 
++ (instancetype)pushEventWithAttributes:(NSDictionary *)attributes {
```

Comment:
Instead of making an API that would need to expand for new, arbitrary things that could all be categorized as "debug" events, let's reuse the existing `debugEventWithAttributes` API and plan on using the attributes for distinction.

I'm saying this also because I think of `push` and `flush` as actions that we want to track in a debug log but not as actual events that we send to the server, if that makes sense.